Shippo/Batch.cs

```diff
@@ -38,6 +38,132 @@ public class Batch : ShippoId {
         [JsonProperty (PropertyName = "object_results")]
         public ObjectResults ObjectResults;
 
+        public class Address
+        {
+            [JsonProperty (PropertyName = "object_purpose")]
+            public string ObjectPurpose;
+
+            [JsonProperty (PropertyName = "name")]
+            public string Name;
+
+            [JsonProperty (PropertyName = "company")]
+            public string Company;
+
+            [JsonProperty (PropertyName = "street1")]
+            public string Street1;
+
+            [JsonProperty (PropertyName = "street2")]
+            public string Street2;
+
+            [JsonProperty (PropertyName = "city")]
+            public string City;
+
+            [JsonProperty (PropertyName = "state")]
+            public string State;
+
+            [JsonProperty (PropertyName = "zip")]
+            public string Zip;
+
+            [JsonProperty (PropertyName = "country")]
+            public string Country;
+
+            [JsonProperty (PropertyName = "phone")]
+            public string Phone;
+
+            [JsonProperty (PropertyName = "email")]
+            public string Email;
+
+            public static Address createForPurchase (String purpose, String name, String street1, String street2, String city,
+                                                     String state, String zip, String country, String phone, String email)
+            {
+                Address a = new Address ();
+                a.ObjectPurpose = purpose;
+                a.Name = name;
+                a.Street1 = street1;
+                a.Street2 = street2;
+                a.City = city;
+                a.State = state;
+                a.Zip = zip;
+                a.Country = country;
+                a.Phone = phone;
+                a.Email = email;
+                return a;
+            }
+        }
+
+        public class BatchShipment
+        {
+            [JsonProperty (PropertyName = "shipment")]
+            public Shipment Shipment;
+
+            public static BatchShipment createForBatchShipments (Shipment shipment)
+            {
+                BatchShipment bs = new BatchShipment ();
+                bs.Shipment = shipment;
+                return bs;
+            }
+        }
+
+        public class Shipment
+        {
+            [JsonProperty (PropertyName = "object_purpose")]
+            public string ObjectPurpose;
+
+            [JsonProperty (PropertyName = "address_from")]
+            public Address AddressFrom;
+
+            [JsonProperty (PropertyName = "address_to")]
+            public Address AddressTo;
+
+            [JsonProperty (PropertyName = "parcel")]
+            public Parcel Parcel;
+
+            public static Shipment createForBatch (String purpose, Address addressFrom, Address addressTo, Parcel parcel)
```

Comment:
The issue here is that the Shipment object defines address_to and address_from as ID strings: [here](https://goshippo.com/docs/reference/bash#shipments). I'd prefer that I create a new class where the types are well defined, instead of casting it from an object